src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -73,7 +73,8 @@ public void setup() {
   public void after() throws Exception {
     TestUtils.dropTable(topicName);
     IngestSdkProvider.setStreamingClientManager(
-        new StreamingClientManager(new HashMap<>())); // reset to clean initial manager
+        StreamingClientManager
+            .resetAndGetEmptyStreamingClientManager()); // reset to clean initial manager
```

Comment:
nice! this cleans it up a lot, lets separate test and prod code by moving it to TestUtils pls

Reply:
Ack!